stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -544,14 +545,13 @@ boot
     return "images/efiboot.img"
 
 
-def mksquashfs_metal(paths, workdir, img_metal, loop_client):
+def mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client):
     """
-    Mounts a copy of the metal image and modifies it accordingly to create a (squashfs) rootfs from its contents for the
-    live ISO.
+    Mounts a copy of the metal image and modifies it accordingly to create a (fstype) rootfs from its contents for the
+    live ISO. fstype must be squashfs or erofs.
 
     Returns the bls entry kernel arguments for the ISO bootloader.
     """
-    squashfs_compression = 'zstd'
     basearch = os.uname().machine
     tmp_squashfs_dir = os.path.join(workdir, 'tmp-squashfs-dir')
```

Comment:
noticed this a bit late, but let's come up with a different name for this variable

```suggestion
    tmp_rootfs_dir = os.path.join(workdir, 'tmp-rootfs-dir')
```

Reply:
will resolve in a followup PR